styles/index.css

```diff
@@ -0,0 +1,56 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+header h1 {
+    margin-left: 500px;
+    margin-top: 20px;
+}
+main {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    height: 580px;
+    width: 900px;
+    margin: 0 auto; 
+    margin-top: 100px;
+
+}
+
+#sunnyday {
+    width: 100%;
+}
+
+.grid-display {
+    height: 100%;
+}
+
+.grid-display:nth-child(1){
+    background-color: gray;
+    
+
```

Comment:
you can remove this white space and any other place that has extra white space to make your code more readable